.github/workflows/ci.yml

```
@@ -23,6 +23,7 @@ env:
   MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -DskipTests -Dmaven.source.skip=true -Dair.check.skip-all"
   MAVEN_GIB: "-P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }}"
   MAVEN_TEST: "-B --strict-checksums -Dmaven.source.skip=true -Dair.check.skip-all --fail-at-end -P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }}"
+  MAVEN_COMPILE_COMMITS: "-B --quiet --strict-checksums --no-snapshot-updates --no-transfer-progress -T C1 -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -Dair.check.skip-all=true -P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }} -pl '!:trino-server-rpm'"
```

```suggestion
  MAVEN_COMPILE_COMMITS: "-B --strict-checksums --quiet -T C1 -DskipTests -Dmaven.source.skip=true -Dair.check.skip-all=true -Dmaven.javadoc.skip=true --no-snapshot-updates --no-transfer-progress -P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }} -pl '!:trino-server-rpm'"
```

(the order makes it easier to see what's actually different from other commands)

Also maybe move it to be just below `MAVEN_FAST_INSTALL`.